packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/PdfViewer.tsx

```diff
@@ -1,11 +1,22 @@
-import React, { SFC, useEffect, useRef, useState, useMemo } from 'react';
-import PdfjsLib from 'pdfjs-dist';
+import React, { FC, useEffect, useRef, useState, useMemo } from 'react';
+import cx from 'classnames';
+import PdfjsLib, {
+  PDFDocumentProxy,
+  PDFPageProxy,
+  PDFPageViewport,
+  PDFRenderTask
+} from 'pdfjs-dist';
 import PdfjsWorkerAsText from 'pdfjs-dist/build/pdf.worker.min.js';
 import { settings } from 'carbon-components';
+import PdfViewerTextLayer, { PdfTextLayerInfo } from './PdfViewerTextLayer';
+
+const { RenderingCancelledException } = PdfjsLib as any;
```

Comment:
Can you add `RenderingCancelledException` as a type in the types file? I think you can just make it of type `Error`, right?